apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -320,6 +337,16 @@ var removeWatermark = function (focusEvent) {
 };
 
 /**
+ * Helper method for determining if a given keyPress event represents a
+ * CLEAN enter press. As in, one without the Shift or Control modifiers.
+ * @param {Event} jqueryEvent
+ * @returns {boolean} true iff the given event represents a clean enter
+ */
+var isCleanEnterPress = function (jqueryEvent) {
```

Comment:
Suggestion: `isUnmodifiedEnterPress` might be more descriptive, since you're checking for the absence of "modifier" keys.